docs/node/daemon/config/settings.mdx

```diff
@@ -208,6 +237,29 @@ tls_key_file = ""
 # Kwild Server hostname
 hostname = ""
 
+#######################################################################
+###                     Extension Configuration                     ###
+#######################################################################
+[app.extensions]
+
+#######################################################################
+###                     Snapshots Configuration                     ###
+#######################################################################
+[app.snapshots]
+
+# Enables snapshots
+enabled = true
+
+# Path to the snapshots directory
+snapshot_dir = "snapshots"
+
+# Heights at which the database is snapshotted.
+snapshot_heights = 10000
```

Comment:
I'm coming back the repetition of the word "snapshot"  This is the `app.snapshots` section, so it's tedious to repeat like `app.snapshots.snapshot_height`.  Also the semantics of this  heights param is still unclear.  This is a "creation interval" or a "period" that schedules snapshot creation, so ideally both the comment description and the setting would hint at that.

Reply:
Sorry, didn't update the docs for this. Updated this on the main.